bootstrap/013_task_queue.sql

```diff
@@ -0,0 +1,54 @@
+
+create or replace procedure task_queue.run_tasks()
+returns varchar
+language sql
+as
+begin
+show warehouses;
+show tasks in schema task_queue;
+let res resultset := (
+with tlist as (
+select "id" as id, "name" as name from table(result_scan(last_query_id())) where "name" <> 'run_tasks'
```

Comment:
Need to pass `database_name` and `schema_name` through, else your query of `account_usage.task_history` is going to potentially overmatch to tasks outside of this database.

Reply:
fixed